docs/manual/contributing.rst

```diff
@@ -74,3 +74,37 @@ Qtile session.
 For any Qtile-specific question on testing, feel free to ask on our `issue
 tracker <https://github.com/qtile/qtile/issues>`_ or on IRC (#qtile on
 irc.oftc.net).
+
+Running tests locally
```

Comment:
Let's not copy paste this, please link from one to the other.

Reply:
Didn't completely copy-paste :P. Which way around? The `contributing.rst` is better, right? Then I'd remove stuff from `CONTRIBUTING.md` and link to the docs.qtile.org page instead.

Reply:
Either way is fine, I just don't want to have to maintain two copies of basically the same text.

Reply:
:heavy_check_mark: